contracts/NestableMultiAsset/README.md

```diff
@@ -500,15 +495,15 @@ Using `RMRKNestingMultiResource` requires custom implementation of minting logic
 The latter is used to nest mint the NFT directly to the parent NFT. If you intend to support it at the minting stage,
 you should implement it in your smart contract.
 
-In addition to the minting functions, you should also implement the burning, transfer and resource management functions if they apply to your use case:
+In addition to the minting functions, you should also implement the burning, transfer and asset management functions if they apply to your use case:
 
 - `_burn(uint256 tokenId)`
 - `transferFrom(address from, address to, uint256 tokenId)`
 - `nestTransfer(address from, address to, uint256 tokenId, uint256 destinationId)`
-- `_addResourceEntry(uint64 id, string memory metadataURI)`
-- `_addResourceToToken(uint256 tokenId, uint64 resourceId, uint64 overwrites)`
+- `_addAssetEntry(uint64 id, string memory metadataURI)`
+- `_addAssetToToken(uint256 tokenId, uint64 assetId, uint64 overwrites)`
 
 Any additional function supporting your NFT use case and utility can also be added. Remember to thoroughly test your
 smart contracts with extensive test suites and define strict access control rules for the functions that you implement.
 
-Happy multiresourceful nesting! 🐣🫧🐣
\ No newline at end of file
+Happy multiassetful nestable! 🐣🫧🐣
```

Comment:
I'd again just use nesting, but if you think it's better to use the name of the lego, its fine 😄